src/store/files/types.ts

```diff
@@ -1,11 +1,13 @@
 import { KlipperFileMeta, Thumbnail } from './types.metadata'
 import { HistoryItem } from '@/store/history/types'
+import { CancelTokenSource } from 'axios'
 
 export type { KlipperFileMeta, Thumbnail }
 
 export interface FilesState {
   uploads: FilesUpload[];
   download: FileDownload | null;
+  fileTransferCancelTokenSource: CancelTokenSource | null;
```

Comment:
This is a mouthful so I'm more than open to naming suggestions!

Reply:
Naming is meh... but honestly, it describes what it is perfectly and that is not visible to the user, so I don't really care! 😁